src/components/eci/Consent.js

```diff
@@ -52,7 +52,7 @@ export default function Register(props) {
               error={!!(errors && errors.certify)}
             >
               <Checkbox
-                inputRef={register}
+                {...register("certify")}
                 color="primary"
                 onChange={handleCheck}
                 name="certify"
```

Comment:
do we need the name if it's in the register?

Reply:
I guess not. I'll remove and test